### PR TITLE
Test mock for edit and clear functions

### PR DESCRIPTION
**Pull request by fabianomag**
Testing the functionality of the edit and clear as mocks

### CODE REVIEW

File: addremove.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { items, addItem, removeItem } from './__mock__/index.js';
 describe('Adding and removing functions', () => {
```
Comment (GzimAsani) [inline addremove.test.js:3]: - [ ] 1 of your tests is failing kindly fix it

Comment (GzimAsani) [reply addremove.test.js:3]: - [ ] Remember to uses mock for functions that used localStorage, DOM, or external APIs and to add more tests

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @fabianofrank  and @CRClothier                                     
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However                                 
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


### PS: For 1 to 1 help you should schedule a meeting with a full-time TSE

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: editclear.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      index: 2,
+      completed: true,
+    };
+    checkItem(item);
+    expect(item.completed).toBeFalsy();
+  });
+
+  test('clears checked items', () => {
+    clearItems(items);
+    expect(items.length).toBe(1);
```
Comment (bellom) [inline editclear.test.js:31]: You might want to check this line, I don't think the length of your `items` is 1, set to the length of your items in the mock file.
I hope this hint helps.

**Comment by bellom** [comment]
Hello 👋,

This is an additional review as a result of the second opinion request.

Sorry guys if the first review wasn't helpful enough, the CR might want you guys to figure it out yourself, I think most of the students master learning that way.

Check out my inline comment above.

Try to make the necessary changes and make sure all your tests pass before submitting for re-review.

Keep rocking 🚀!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**